pymc3/smc/sample_smc.py

```diff
@@ -223,10 +231,25 @@ def sample_smc(
     trace.report.nsteps = nsteps
     trace.report._t_sampling = time.time() - t1
 
+    if compute_convergence_checks or return_inferencedata:
+        ikwargs = dict(model=model)
+        idata = to_inference_data(trace, **ikwargs)
```

Comment:
For `pm.sample` we have a kwarg to pass additional parameters such as `log_likelhood=False` for the conversion. This is important for some models that are expensive to evaluate. Without this the progress bar can appear to get stuck at 100 % while the conversion re-evaluates the model at every iteration, potentially taking as long as the original MCMC.

Reply:
Yeah, actually in our case we will just use the pre-computed `log_likelihood` since we have it from the sampler. I just have to figure out how to initialize the idata with it, so it doesn't try to recompute it. Are there other optional arguments that the user might want to pass besides this?